app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, first_file, second_file, *_ = command.split()
+
+    if first_file == second_file or command.split(" ")[0] != "cp":
```

Comment:
you can use existed `command_name` instead of running split again

Reply:
@LLkaia Oh sorry. I didn't notice that!